src/lib/db/access-store.ts

```diff
@@ -217,7 +217,7 @@ export class AccessStore implements IAccessStore {
         userPermissionQuery = userPermissionQuery.union((db) => {
             db.select(
                 'project',
-                'permission',
+                'p.permission',
```

Comment:
I guess in 5.6 role_permission table does not have the permission column, right? That's why you chose `p.permission`  instead of `rp.permission`

Reply:
Correct 👍 The reason why I opted for `rp` in [#5409](https://github.com/Unleash/unleash/pull/5409) was because we may want to eventually drop the join. But since `p.permission` exists in both *before* and *after* the migration, it would be the best option.